spec/flows/household_flow_spec.rb

```diff
@@ -7,24 +7,36 @@
     fill_in_applicant_screen(partner: "Yes")
     fill_in_forms_until(:outgoings)
```

Comment:
Given that this spec is now the official definition of the expected household flow, I think there's a case for replacing `fill_in_forms_until` with an explicit list of `fill_in_x_screen`, `fill_in_y_screen`. Quite a lot has changed between the applicant screen and the outgoings screen and it would be good to list out what that new flow looks like.